crates/sui-network/src/state_sync/builder.rs

```diff
@@ -77,12 +77,14 @@ where
         let metrics = metrics.unwrap_or_else(Metrics::disabled);
 
         let (sender, mailbox) = mpsc::channel(config.mailbox_capacity());
-        let (checkpoint_event_sender, _reciever) =
+        let (checkpoint_event_sender, _receiver) =
             broadcast::channel(config.synced_checkpoint_broadcast_channel_capacity());
+        let (checkpoint_watcher_sender, _receiver) = tokio::sync::watch::channel(());
```

Comment:
You can also consider `tokio::sync::Notify` instead of `()` channel (other checkpoint code uses it extensively)